openshift_pipelines/understanding-openshift-pipelines.adoc

```diff
@@ -0,0 +1,28 @@
+[id="understanding-openshift-pipelines"]
+= Understanding OpenShift Pipelines
+include::modules/openshift-pipelines-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: understanding-openshift-pipelines
+toc::[]
+
+:FeatureName: OpenShift Pipelines
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+{op-title} is a cloud-native, continuous integration and continuous delivery (CI/CD) solution based on Kubernetes resources. It uses link:https://tekton.dev[Tekton] building blocks to automate deployments across multiple platforms by abstracting away the underlying implementation details. Tekton introduces a number of standard link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Custom Resource Definitions (CRDs)] for defining CI/CD pipelines that are portable across Kubernetes distributions.
+
+[id="op-key-features"]
+== Key features
+
+* {op-title} is a serverless CI/CD system that runs Pipelines with all the required dependencies in isolated containers.
+* {op-title} are designed for decentralized teams that work on microservice based architecture.
+* {op-title} use standard CI/CD pipeline definitions that are easy to extend and integrate with the existing Kubernetes tools, enabling you to scale on-demand.
+* You can use {op-title} to build images with Kubernetes tools such as Source-to-Image (S2I), Buildah, Buildpacks, and Kaniko that are portable across any Kubernetes platform.
+* You can use the {product-title} Developer Console to create Tekton resources, view logs of Pipeline runs, and manage pipelines under your {product-title} project namespaces.
```

Comment:
Not a part of your changes, but can you please remove 'project', both namespaces and project almost mean the same thing.
Under/in?